substrate/frame/referenda/src/lib.rs

```diff
@@ -225,8 +226,27 @@ pub mod pallet {
 	#[pallet::extra_constants]
 	impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		#[pallet::constant_name(Tracks)]
```

Comment:
Maybe add a doc comment saying this constant may be inaccurate if the tracks are dynamic in the runtime.

Reply:
What's your suggestion of the comment? I'm thinking about something like:

```rs
/// A list of tracks.
///
/// Note: if the tracks are dynamic, the fetch performance might be inaccurate.
```

Reply:
Maybe this?

The point is that metadata are released for a runtime spec version. Those static metadata will be outdated if value is changed in storage in the meanwhile.

/// A list of tracks.
///
/// Note: if the tracks are dynamic, the value in the static metadata might be inaccurate.

Reply:
Done.